generateMojang.py

```diff
@@ -198,6 +198,21 @@ def adapt_new_style_arguments(arguments):
             pprint(arg)
     return " ".join(foo)
 
+def adapt_new_style_arguments_to_traits(arguments):
+    foo = []
+    # we ignore the jvm arguments entirely.
+    # grab the object, log the errors
+    for arg in arguments.game:
+        if not isinstance(arg, str):
+            try:
+                for rule in arg["rules"]:
+                    for k,v in rule["features"].items():
+                        if rule["action"] == "allow" and v:
+                            foo.append(f"feature:{k}")
```

Comment:
I think we should only add traits for features we *know* about. Otherwise it just pollutes the metadata with unused stuff

Reply:
Should I also drop the prefix or keep it?